frontend/src/containers/SignUp.js

```diff
@@ -13,6 +13,11 @@ class SignUp extends Component {
   onSubmit(e) {
     e.preventDefault();
     this.setState({ isLoading: true });
+    setTimeout(() => {
```

Comment:
this is a pretty weird solution, can you explain to me what you think was going wrong and how this fixes it?

Reply:
You are right. `setTimeout` is not necessary, just wanted to show the `loading` effect of the button because backend in localhost is too fast. The redirect is part of solving the issue. If we provide a custom element for a form, we should provide some way to programmatically handle the form success, with a callback or something. Since this is just a demo, I forced the "callback" to the "ConsentSuccess" component. In the future, we should provide something like this, the same way Auth0 does.

Reply:
Lets chat about this in the meeting tomorrow I want to make sure we aren't band aiding a deeper issue here by forcing a redirect with window.location